go.mod

```diff
@@ -6,33 +6,30 @@ require (
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/PuerkitoBio/purell v1.0.0 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2 // indirect
-	github.com/davecgh/go-spew v0.0.0-20151105211317-5215b55f46b2
+	github.com/davecgh/go-spew v1.1.0
 	github.com/emicklei/go-restful v0.0.0-20170410110728-ff4f55a20633
 	github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
 	github.com/go-openapi/jsonpointer v0.0.0-20160704185906-46af16f9f7b1 // indirect
 	github.com/go-openapi/jsonreference v0.0.0-20160704190145-13c6e3589ad9 // indirect
 	github.com/go-openapi/spec v0.0.0-20160808142527-6aced65f8501
 	github.com/go-openapi/swag v0.0.0-20160704191624-1d0bd113de87 // indirect
 	github.com/golang/protobuf v0.0.0-20161109072736-4bd1920723d7
-	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367
+	github.com/google/gofuzz v1.0.0
 	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d
-	github.com/json-iterator/go v0.0.0-20180612202835-f2b4162afba3
+	github.com/json-iterator/go v1.1.6
 	github.com/mailru/easyjson v0.0.0-20160728113105-d5b7844b561a // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v0.0.0-20180320133207-05fbef0ca5da // indirect
 	github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d
 	github.com/onsi/ginkgo v0.0.0-20170829012221-11459a886d9c
 	github.com/onsi/gomega v0.0.0-20170829124025-dcabb60a477c
-	github.com/pmezard/go-difflib v0.0.0-20151028094244-d8ed2627bdf0 // indirect
 	github.com/spf13/pflag v0.0.0-20170130214245-9ff6c6923cff
-	github.com/stretchr/testify v0.0.0-20151208002404-e3a8ff8ce365
+	github.com/stretchr/testify v1.3.0
 	golang.org/x/net v0.0.0-20170114055629-f2499483f923 // indirect
 	golang.org/x/sys v0.0.0-20170830134202-bb24a47a89ea // indirect
 	golang.org/x/text v0.0.0-20160726164857-2910a502d2bf // indirect
 	golang.org/x/tools v0.0.0-20181011042414-1f849cf54d09 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 	k8s.io/gengo v0.0.0-20190128074634-0689ccc1d7d6
 	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e
+	sigs.k8s.io/structured-merge-diff/v3 v3.0.0-20200116222232-67a7b8c61874
```

Comment:
This tag doesn't exist, does it? Because to be honest, v3 has not stabilized yet :-)

Reply:
No, there is no v3.0.0 tag yet, this was generated when I changed the import line and called `go mod tidy`